srml/aura/src/lib.rs

```diff
@@ -167,7 +242,15 @@ decl_storage! {
 }
 
 decl_module! {
-	pub struct Module<T: Trait> for enum Call where origin: T::Origin { }
+	pub struct Module<T: Trait> for enum Call where origin: T::Origin { 
+		/// Report equivocation.
+		fn report_equivocation(
+			_origin,
+			_equivocation_proof: AuraEquivocationProof<T::Header, T::Signature>
+		) {
+			// This is the place where we will slash.
```

Comment:
This needs an issue filed for it, along with a FIXME and the issue number in the code.